iphone/Resources/alloy/widgets/custom/controllers/widget.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
```

Comment:
This seems to be part of your local testing environment and should not be committed.

Reply:
This was intentionally committed so you guys could work from the same example quickly. It is intended to be reverted.